share/p2p/shrexeds/options.go

```diff
@@ -83,6 +83,6 @@ func WithConcurrencyLimit(concurrencyLimit int) Option {
 	}
 }
 
-func protocolID(protocolSuffix string) protocol.ID {
-	return protocol.ID(fmt.Sprintf("%s%s", protocolPrefix, protocolSuffix))
+func protocolID(networkID string) protocol.ID {
+	return protocol.ID(fmt.Sprintf("%s%s", networkID, protocolPrefix))
```

Comment:
let's call `protocolPrefix` just `protocolString` or something and then remove trailing `/`

networkID will also need a leading `/` no?

Reply:
Why don't you like networkID here?  Prefix/suffix is smth abstract, but here we are explicitly setting networkID

Reply:
Rene is not against networkID and want protocolPrefix to be renamed

Reply:
oh....sorry
Was responded from phone

Reply:
> networkID will also need a leading / no?

iirc, no